frontend/src/components/dashboard/page_timings/PageTimingsTable.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+    <VContainer
+        p-0
+        class="users-table"
+    >
+        <VRow
+            class="header my-3"
+            fluid
+        >
+            <VCol>
+                <VSelect
+                    class="option-select"
+                    prefix="Show"
+                    :items="options"
+                    flat
+                    v-model="selected"
+                    @change="changeSelect"
```

Comment:
it's quite bad practice to use model event alongside with `v-model`, also having local state, that basically affect on parents data can lead  to inconsistent state, when your dropdown will show value, that will be totally different from value that was used for grouping